src/plugins/footnotes/footnotes-utils.ts

```diff
@@ -78,6 +81,7 @@ export const buildTableFootnoteLabels = (node: ManuscriptNode) => {
   let index = 0
 
   findChildrenByType(node, schema.nodes.inline_footnote)
+    .filter(({ node }) => !(isRejectedInsert(node) || isDeleted(node)))
```

Comment:
We want to keep the deleted `inline-footnote` nodes. It's expected for the label to stay there with a line through.